bucket/granted.json

```
@@ -0,0 +1,29 @@
+{
+    "version": "0.21.1",
+    "homepage": "https://github.com/common-fate/granted",
+    "description": "Granted is a command line interface (CLI) application which simplifies access to cloud roles and allows multiple cloud accounts to be opened in your web browser simultaneously.",
+    "license": "MIT",
+    "depends": ["aws"],
+    "architecture": {
+        "32bit": {
+            "url": "https://releases.commonfate.io/granted/v0.21.1/granted_0.21.1_windows_i386.zip",
+            "hash": "41cd7a2d3aff65fb38d81de6ba76f72c0a6c773a82d0f11b8b07228f0b5c52f7"
+        },
+        "64bit": {
+            "url": "https://releases.commonfate.io/granted/v0.21.1/granted_0.21.1_windows_x86_64.zip",
+            "hash": "91ce76167734d3a56182bff4f99a2d80ac9165d4a0c642ba95c85c1f0aa3e25f"
+        }
+    },
+    "bin": ["assume.ps1", "granted.exe"],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "32bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_i386.zip"
+            },
+            "64bit": {
+                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_amd64.zip"
+            }
```

```suggestion
            },
            "arm64": {
                "url": "https://releases.commonfate.io/granted/v$version/granted_$version_windows_arm64.zip"
            }
```

Add arm64 version as well.